js/views/pageNavVw.js

```diff
@@ -479,6 +480,14 @@ module.exports = Backbone.View.extend({
     this.currentWindow.reload();
   },
 
+  addressBarFocus: function(e){
+    // on inital focus of input, select all text (this makes it easier to copy or delete the text)
+    $(this.$el).one('mouseup', function () {
+      $('#addressBar').select()
+      return false;
+    }).select();
```

Comment:
Ah, I see what you were trying to do.

I'll switch the "on" back to "one". The rest of the code isn't needed.